### PR TITLE
[cli] added micro benchmarking for tp

### PR DESCRIPTION
**Pull request by YuliangLiu0306**

### CODE REVIEW

**Comment by FrankLeeeee** [comment]
Hi @YuliangLiu0306 , CodeFactor has found some minor issues in your code, can you fix it?

**Comment by YuliangLiu0306** [comment]
> Hi @YuliangLiu0306 , CodeFactor has found some minor issues in your code, can you fix it?



> Hi @YuliangLiu0306 , CodeFactor has found some minor issues in your code, can you fix it?

Done.

File: setup.py [hunk 1]
```diff
@@ -213,6 +213,10 @@ def cuda_ext_helper(name, sources, extra_cuda_flags, extra_cxx_flags=[]):
     ext_modules=ext_modules,
     cmdclass={'build_ext': BuildExtension} if ext_modules else {},
     install_requires=fetch_requirements('requirements/requirements.txt'),
+    entry_points='''
+        [console_scripts]
+        colossal=colossalai.scripts.cli:cli
```
Comment (FrankLeeeee) [inline setup.py:218]: Is there any purpose of creating two-level directories? I think the norm is to have a folder directly under the main repo, e.g. `/colossalai/cli`

File: colossalai/benchmark/run.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import torch
+import inspect
+import os
+import subprocess
+import sys
```
Comment (FrankLeeeee) [inline colossalai/benchmark/run.py:5]: Should we put the benchmark files into the cli folder as well? I think this module is not gonna be used by the user directly, so better to hide it from the user.

**Review by FrankLeeeee — APPROVED** [review]

**Pull request merged**